terraform/install.sh

```diff
@@ -260,7 +260,7 @@ installMonitoring() {
   log "Creating monitoring examples (dashboards, uptime checks, alerting policies, etc.)..."
   pushd monitoring/
   terraform init -lockfile=false
-  terraform apply --auto-approve -var="project_id=${project_id}" -var="external_ip=${external_ip}" -var="project_owner_email=${acct}" -var="zone=${CLUSTER_ZONE}"
+  terraform apply --auto-approve -var="project_id=${project_id}" -var="external_ip=${external_ip}" -var="project_owner_email=${acct}" -var="zone=${CLUSTER_ZONE}" -var="skip_ratingservice=${skip_ratingservice:-false}"
```

Comment:
do we need to add `skip_loadgen` here too?

Reply:
yes, we need to disable ratingservice dashboards when ratingservice is disabled, right?

Reply:
I mean that this call does not set `skip_loadgen` parameter. My question was do we need to add the `--skip_loadgen` in addition to `skip_ratingservice`?

Reply:
Oh sorry, misread that. No, slip_loadgen isn't needed because there are no monitoring dashboards for the loadgen cluster